awx/playbooks/action_plugins/project_archive.py

```diff
@@ -62,10 +69,19 @@ def run(self, tmp=None, task_vars=None):
             try:
                 is_dir = member.is_dir()
             except AttributeError:
-                is_dir = member.isdir()
+                try:
+                    is_dir = member.isdir()
+                except AttributeError:
+                    is_dir = member.filename[-1] == '/'  # py2 compat for ZipInfo
```

Comment:
python2's `zipfile` doesn't have this, but this is what py3 does, so I just copied it here <shrug>